libpod/options.go

```diff
@@ -181,7 +181,7 @@ func WithStaticDir(dir string) RuntimeOption {
 // WithHooksDir sets the directory to look for OCI runtime hooks config.
 // Note we are not saving this in database, since this is really just for used
 // for testing.
-func WithHooksDir(hooksDir string, dirNotExistFatal bool) RuntimeOption {
+func WithHooksDir(hooksDir string) RuntimeOption {
```

Comment:
I want to keep `dirNotExistFatal` for anything explicitly set by the caller (i.e. not hard-coded into libpod).  More discussion on this in b76c150d3aa.